gui/pages/user_add.go

```diff
@@ -202,6 +202,7 @@ func (page *UserAddPage) onPasswordChange(entry *gtk.Entry) {
 	}
 
 	password := getTextFromEntry(page.password)
+
```

Comment:
Either update the copyright notice to 2020 or better yet, revert adding this blank line and leave the file unchanged.

Reply:
Done